amf-aml/shared/src/main/scala/amf/plugins/document/vocabularies/AMLInstancePlugin.scala

```diff
@@ -0,0 +1,26 @@
+package amf.plugins.document.vocabularies
+
+import amf.client.remod.amfcore.plugins.{HighPriority, PluginPriority}
+import amf.client.remod.amfcore.plugins.parse.{AMFParsePlugin, ParsingInfo}
+import amf.core.Root
```

Comment:
What does this have to do with the PR?

Reply:
Nothing, but as the Nico's PR is not merged yet, I added to build the skeleton of some AMLConfiguration methods.